pkg/syncer/status/status_process_test.go

```diff
@@ -588,12 +586,12 @@ func setupServersideApplyPatchReactor(toClient *dynamicfake.FakeDynamicClient) {
 	})
 }
 
-func setupWatchReactor(resource string, fromClient *dynamicfake.FakeDynamicClient) chan struct{} {
+func setupWatchReactor(resource string, client *dynamicfake.FakeDynamicClient) chan struct{} {
```

Comment:
why unqualify the client?

Reply:
Because this method is now use on both the fromClient and the toClient of the testCase, since upstream informers are also used to get upstream objects now.

Reply:
ack